### PR TITLE
Keyword col

### PR DESCRIPTION
**Pull request by qubixes**

### CODE REVIEW

File: asreview/datasets.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    link = "https://zenodo.org/record/1162952#.XIVBE_ZFyVR"
+    authors = ["Tracy Hall", "Sarah Beecham", "David Bowes", "David Gray",
+               "Steve Counsell"]
+    year = 2012
+    license = "CC-BY Attribution 4.0 International"
+    topic = "Software Fault Prediction"
+
+
+class ExampleDataSetGroup():
+    data_sets = [
```
Comment (J535D165) [inline asreview/datasets.py]: self? Not sure. Especially useful to make an isolated ExampleDataSetGroup?

Do we need an append method as well?

Comment (qubixes) [reply asreview/datasets.py]: I'm not quite sure what you mean. I added an append method, though I don't think it needs to be used really. Maybe with the update it makes more sense?

Comment (J535D165) [reply asreview/datasets.py]: No problem, if this works with an extension that adds datasets, I'm fine.

**Comment by J535D165** [comment]
Closes #182

**Pull request merged**